health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/MultiSelectDropdown.js

```
@@ -0,0 +1,1010 @@
+  const boundaryType = alreadyQueuedSelectedState.find((item) => item.propsData[1] !== null)?.propsData[1]?.type;
+  const { t } = useTranslation();
+  function removeFromDummySelected(...props) {
+    const updatedDummySelectedData = dummySelected.filter((item) => item.code !== props?.[1]?.code);
+    setDummySelected(updatedDummySelectedData);
+  }
+  function handleSelected(value) {
+    let res = [];
+    value &&
+      value?.map((ob) => {
+        res.push(ob?.[1]);
+      });
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)